nsxt/resource_nsxt_policy_security_policy.go

```diff
@@ -96,28 +70,58 @@ func policySecurityPolicyBuildAndPatch(d *schema.ResourceData, m interface{}, co
 		obj.Revision = &revision
```

Comment:
we shouldn't set revision for create flow even if rules are present

Reply:
Good catch. Done.